public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -134,6 +136,30 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     this.showTopConcepts = true;
   }
 
+  private setDomain() {
```

Comment:
this sets the domain obj in local storage based on the route in the url.
it does a search across all domains but,
it will only run if there isn't already a domain obj.